tests/integration/debug/cmd_debug_test.go

```diff
@@ -32,8 +32,12 @@ var _ = Describe("odo debug command serial tests", func() {
 	})
 
 	It("should auto-select a local debug port when the given local port is occupied", func() {
+		projName := helper.GetCliRunner().CreateRandNamespaceProjectOfLength(5)
```

Comment:
Could you please add it in `JustBeforeEach`. Mostly `It` spec consists only the test steps.

Reply:
done